media/CfgEditor/index.js

```diff
@@ -101,6 +104,82 @@ function main() {
   postMessageToVsCode({ type: "requestDisplayCfg" });
 }
 
+function setDefaultValues(name) {
+  const optimizationUsed = document.getElementById("checkboxOptimize").checked;
+
+  let importedPath = name;
+  let optimizedPath = importedPath + ".opt";
+  let quantizedPath = optimizationUsed
+    ? optimizedPath + ".Q"
+    : importedPath + ".Q";
```

Comment:
using capital letters in filename makes typing annoying.
how about using `q8` for `uint8` and `q16` for `int16` from `DefaultQuantQuantizedDtype` ?
this convention is used for our internal release.

Reply:
Ok. I'll try!